Source/Pawnmorphs/Esoteria/HPatches/PawnCompPatches.cs

```diff
@@ -103,6 +104,12 @@ private static void GiveSapientAnimalsNeeds(Pawn_NeedsTracker __instance, Pawn _
                     __result = Need_Control.IsEnabledFor(___pawn);
                     return;
                 }
+                //Need_Control.EnabledRaces.Contains(___pawn.def); Can't use it because it's private
```

Comment:
Wouldn't this be similar to using the "IsEnabledFor" function called just a bit further up?

Reply:
Yes. But Need_Control is only called currently if the investigated need is the Sapience Need, instead of checked for Every Need Call.

Checked Androids mod, and it doesn't appear like they have any special compatibility options set, warranting a special Blacklist.